src/hypothesis/extra/django/models.py

```diff
@@ -95,6 +95,33 @@ def validate(value):
     return validate
 
 
+safe_letters = string.ascii_letters + string.digits + '_-'
+
+domain = st.builds(
+    lambda x, y: '.'.join(x + [y]),
+    st.lists(st.text(safe_letters, min_size=1)), st.sampled_from([
```

Comment:
This strategy will sometimes produce bare TLDs, and regularly. That seems like a bug. e.g.

```python
for _ in range(25):
    d = domain.example()
    if '.' not in d:
        print(d)
```

returns 

```
com
net
net
com
com
biz
```

Running a similar test with the old strategy doesn’t turn up any email addresses that didn’t have a multi-part domain name after the `@` symbol:

```python
for _ in range(1000):
    e = ff(u'email').example()
    if '.' not in e.split('@')[1]:
        print(e)
```

doesn’t return anything for me.

Should there be a `min_size=1` for `st.lists()` as well?